app/models/admin_ability.rb

```diff
@@ -0,0 +1,241 @@
+class AdminAbility
+  include CanCan::Ability
+
+  def initialize(user)
+    # Order Abilities
+    # (Check https://github.com/CanCanCommunity/cancancan/wiki/Ability-Precedence)
+    # Check roles of user, using rolify. Role name is *case sensitive*
+    # user.is_organizer? or user.has_role? :organizer
+    # user.is_cfp_of? Conference or user.has_role? :cfp, Conference
+    # user.is_info_desk_of? Conference
+    # user.is_volunteers_coordinator_of? Conference
+    # user.is_attendee_of? Conference
+    # The following is wrong because a user will only have 'cfp' role for a specific conference
+    # user.is_cfp? # This is always false
+
+    user ||= User.new
+    signed_in_with_roles(user)
+  end
+
+  def common_abilities_for_all
```

Comment:
These abilities are defined for the not_signed_in users as well, and here we are not adding anything new/more. So do we really need this whole function?